docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,196 @@
+| **Complexity**     | Simpler to implement but requires monitoring.   | Computationally intensive and harder to build.                          |
+| **Use Cases**      | Broad compatibility with existing applications. | Best for high-security applications where instant finality is critical. |
+
+Both approaches serve different needs and trade-offs, depending on the balance between scalability,
+security, latency, and flexibility required.
+
+### Zero-knowledge Rollups
+
+Zero-knowledge rollups (often called ZK rollups) are a _non-interactive_ method that utilizes
+zero-knowledge proofs to compute the validity of a particular set of state changes. Whereas
+[optimistic rollups](#optimistic-rollups) rely on fraud proofs, ZK rollups rely on cryptographic
+validation in the form of ZK proofs.
+
+Zero-knowledge rollups are significantly faster in finalization, as the cryptographic validity proof
+handles the nuance of ensuring a rollup is valid. However, the ZK rollups often suffer from a
+performance perspective due to the complexity involved in proof generation and difficult
+implementation into resource-constrained environments. Because Turing completeness is also
+challenging to achieve due to this computational overhead, their ability to be generalized (in terms
+of blockspace) is reduced. However, the generation of zk proofs is becoming useful in niche
+applications. For example, [Hyperbridge](./learn-bridges.md#bridge-comparison) is a ZK rollup on
+Polkadot that serves as a scalable trustless bridge producing a single zk proof for multiple
+blockchains that can be instantly verified on Polkadot.
+
+### Optimistic Rollups
+
+Optimistic rollups are an interactive scaling method for L1 blockchains. They assume
+_optimistically_ that every proposed transaction is valid by default.
+
+In the case of mitigating potentially invalid transactions, optimistic rollups introduce a
+_challenge period_ during which participants may challenge a suspect rollup. A fraud-proving scheme
+can be in place to allow for several _fraud proofs_ to be submitted. Those proofs could make the
+rollup valid or invalid. During the challenge period, state changes may be disputed, resolved, or
+included if no challenge is presented (and the required proofs are in place).
+
+Optimistic rollups are often used in the Ethereum ecosystem. [Optimism](https://www.optimism.io/)
+and [Arbitrium](https://bridge.arbitrum.io/) are an example of optimistic EVM-based rollups.
+
+### Polkadot Rollups
+
+Polkadot Rollups work similarly to optimistic rollups. They are an interactive method with fraud
+proof mechanism. Very much like optimistic rollups and zk rollups are secured by Ethereum, Polkadot
+rollups are secured by the Polkadot Relay Chain. The checking and fraud proof mechanics are
+implemented on Polkadot, natively. [collators](./learn-collator.md), are in principle similar to
+sequencers, as they pass data with a proof-of-validity (PoV) function for liveness and communication
```

Comment:
maybe worth it to add "block witness" or mention that Polkadot is a "stateless client" of parachains as these are more familiar terms in ethereum land

Reply:
yep, added